airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorReleases.yaml

```diff
@@ -1,4 +1,3 @@
----
 "$schema": http://json-schema.org/draft-07/schema#
 "$id": https://github.com/airbytehq/airbyte/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorReleases.yaml
 title: ConnectorReleases
```

Comment:
This comment is probably a bit of of scope: can `breakingChanges` be an empty object?
In other word, can this model title be renamed to `ConnectorBreakingChanges` if each entry denotes a breaking change in a specific version?

Reply:
This model is the model for `releases:` which has the `breakingChanges` key underneath it. `breakingChanges` is the key for which each entry denotes a breaking chnage in a specific version- that one is called `ConnectorBreakingChanges`. 

Looks like both `ConnectorBreakingChanges` and `ConnectorReleases` have the same description- it's incorrect for `ConnectorReleases`. Good catch!

Reply:
Updated the description in [26c1f90](https://github.com/airbytehq/airbyte/pull/33111/commits/26c1f9095378ffa54cd906190b4863c69e9c9dc1)

Reply:
with regards to 

> can breakingChanges be an empty object?

I worry it is probably the same issue as [here](https://github.com/airbytehq/airbyte/pull/33111#discussion_r1424337577)...